types/material-ui/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for material-ui v0.17.5
+// Type definitions for material-ui v0.17.51
```

Comment:
for future reference the first two parts of the version are the only ones honored. consider adding a tslint.json file to this package, and that would flag these issues.